jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsSubsystemDefinition.java

```diff
@@ -85,9 +85,6 @@ public void registerAdditionalRuntimePackages(ManagementResourceRegistration res
                     RuntimePackageDependency.optional(RESTEASY_MULTIPART.getName()),
                     RuntimePackageDependency.optional(RESTEASY_CRYPTO.getName()),
                     RuntimePackageDependency.optional(JACKSON_DATATYPE_JDK8.getName()),
-                    RuntimePackageDependency.optional(JACKSON_DATATYPE_JSR310.getName()),
-                    // The following ones are optional dependencies located in org.jboss.as.jaxrs module.xml
-                    // To be provisioned, they need to be explicitly added as optional packages.
-                    RuntimePackageDependency.optional("org.jboss.resteasy.resteasy-spring"));
```

Comment:
I don't think this was required, but I could use some validation from someone that understands Galleon more than I do. From my understanding the module will be configured in WildFly Preview because it's defined there and there is a task that runs to copy the actual libraries into the modules directory.